modules/builtin/src/backend/variables/boolean.ts

```diff
@@ -0,0 +1,47 @@
+import { BoxedVarContructor, BoxedVariable } from 'botpress/sdk'
+
+class BoxedBoolean implements BoxedVariable<boolean> {
+  public static type = 'boolean'
```

Comment:
Does this make renference to another metadata type that we will pass at the module entry? Or do we want to define the needed params and such within the boxed variable type (as a static variable)? I think it would make sense to provide the variable type object at the module entry, and have this object give us the boxed type if that makes sense. Unless we consider that variable types and boxed types aren't 1:1

Reply:
Ok that makes sense. I suppose we'll want to describe what kind of menu components we want in this config. I think this has been done before so I'll base myself on the existing implementation.